libmamba/include/mamba/core/output.hpp

```diff
@@ -170,7 +170,7 @@ namespace mamba
     {
     public:
 
-        MessageLogger(const char* file, int line, log_level level);
+        MessageLogger(const char* file, log_level level);
```

Comment:
Can you explain why remove the line here?

Reply:
`m_line` is not used anywhere, we're not logging the line number

Reply:
It doesn't give a warning (now), but it's a good point, I will remove `m_file` as well - it doesn't change any behaviour.